docs/source/error_measures.rst

```diff
@@ -23,6 +23,8 @@ Example::
 
 .. autoclass:: RootMeanSquaredError
 
+.. autoclass:: NormalisedRootMeanSquaredError
```

Comment:
We're doing everything alphabetical now, so this needs to go up before ProbabilityBasedError !

Reply:
:D OK! I thought about it, but then it didn't look alpahbetically ordered, so I put it to where I thought the most sensible place. Sure, I can move it!